planning/scenario_planning/lane_driving/motion_planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -225,12 +209,13 @@ void ObstacleStopPlannerNode::pathCallback(
 
     Eigen::Matrix4f affine_matrix =
       tf2::transformToEigen(transform_stamped.transform).matrix().cast<float>();
-    sensor_msgs::PointCloud2 transformed_obstacle_ros_pointcloud;
-    pcl_ros::transformPointCloud(
-      affine_matrix, *obstacle_ros_pointcloud_ptr_, transformed_obstacle_ros_pointcloud);
+    pcl::PointCloud<pcl::PointXYZ>::Ptr obstacle_ros_pointcloud_pcl_ptr_(
```

Comment:
Should this variable name exclude "ros" as it is no longer a ros type?

Reply:
you are right, I've updated the variable name in the latest commit